packages/core/src/local-admin-api.ts

```diff
@@ -0,0 +1,37 @@
+import { AdminApi, DiagnosticsLogger, LogStyle } from '@ceramicnetwork/common'
+import { StreamID } from '@ceramicnetwork/streamid'
+import { LocalIndexApi } from './indexing/local-index-api.js'
+
+/**
+ * AdminApi for Ceramic core.
+ */
+export class LocalAdminApi implements AdminApi {
+  constructor(
+    private readonly indexApi: LocalIndexApi,
+    private readonly logger: DiagnosticsLogger
+  ) {}
+
+  async addModelsToIndex(modelsIDs: Array<StreamID>): Promise<void> {
+    this.logger.log(LogStyle.info, `Local Admin Api will add model ids to index ${modelsIDs}`)
+    await this.indexApi.indexModels(modelsIDs)
+    this.logger.log(LogStyle.info, `Local Admin Api did add model ids to index ${modelsIDs}`)
+  }
+
+  getIndexedModels(): Promise<Array<StreamID>> {
+    return Promise.resolve(this.indexApi.indexedModels() ?? [])
+  }
+
+  async removeModelsFromIndex(modelsIDs: Array<StreamID>): Promise<void> {
+    this.logger.log(LogStyle.info, `Local Admin Api will remove model ids to index ${modelsIDs}`)
+    await this.indexApi.stopIndexingModels(modelsIDs)
+    this.logger.log(LogStyle.info, `Local Admin Api did remove model ids to index ${modelsIDs}`)
+  }
+
+  async replaceModelsInIndex(modelsIDs: Array<StreamID>): Promise<void> {
```

Comment:
Non-actionable. I have a bad feeling about it. This `replace` thing might be a source of confusion.